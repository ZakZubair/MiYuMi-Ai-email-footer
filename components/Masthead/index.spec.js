import React from 'react'
import { shallow } from 'enzyme'
import Masthead from './index'
import serializer from 'enzyme-to-json/serializer'
expect.addSnapshotSerializer(serializer)

const links = [
  {
    href: 'https://github.com/ZakZubair/',
    text: 'Zak Zubair'
  },
  {
    href: 'http://MiYuMi.Ai',
    text: 'MiYuMi.Ai'
  }
]

test('Masthead snapshot test', () => {
  const component = shallow(<Masthead title='Zander' links={links} />).toJSON()
  expect(component).toMatchSnapshot()
})

it('should render a Masthead with title', () => {
  const wrapper = shallow(<Masthead title='Events' />)
  expect(wrapper.instance().props.title).toEqual('Events')
})
