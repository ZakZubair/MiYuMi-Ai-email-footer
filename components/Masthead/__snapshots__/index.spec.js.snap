// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Masthead snapshot test 1`] = `
<div>
  <h1>
    Zak Zubair
  </h1>
  <a
    href="https://github.com/ZakZubair/"
  >
    Zak Zubair
  </a>
  <a
    href="http://MiYuMi.Ai"
  >
    MiYuMi.Ai
  </a>
</div>
`;
